docs/UsingPerkStore.rst

```diff
@@ -0,0 +1,106 @@
+.. _UsingPerkStore:
+
+####################
+Using the Perkstore
+####################
+
+The perkstore is an online store available in the platform. 
+You are able to buy the available products or sell goods or services using the cauris as a currency.
+
+How to access to Perkstore
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can access to perkstore from the left navigation. By clicking on the appropriate icon, you will display all the available products in the store.
+You can see your wallet balance in the toolbar as well as a search bar and a filter button that allow you to look for a specific product.
+
+|image0|
+
+You can filter the display of articles by:
+- Disable products
+- Soldout products
```

Comment:
```suggestion
- Sold out products : products that are no longer in store
```

Reply:
The products that are no longer available in stock.
WDYT?